FAQ's.md

```diff
@@ -0,0 +1,30 @@
+**Q: What is ADSdroid ?**
+
+A: It is the unofficial AllDataSheet Android Application. It gives her users the privilege of searching and downloading electronics tools in pdf format, for easier and faster access 
+
+**Q: Does it requires internet access ?**
+
+A: Absolutely. It requires internet connection, for searching and also for downloading files in PDF format.
+
+**Q: How do I use it ?**
+
+A: Very simple. Just input a valid electrical terms in the search box and click on the search button 
+
+**Q: What is the drop down-button for ?**
+
+A: When you click on the drop-down button, you'll see some options(included, start with, end with and match), these are used to make your search results more precise 
```

Comment:
please leave a space before the opening parentheses, like `options (included`

Reply:
This could be a bit more useful if you'd describe the four options in a bulleted list, maybe with some examples like the `ATMEGA168` (a typical component whose datasheet someone would search for using this app) could be matched with east of the options, like it includes `MEG`, ends with `MEGA168`, starts with `ATMEGA`, and only matches `ATMEGA168`.